src/legacy/status_im/utils/logging/core.cljs

```diff
@@ -158,22 +163,8 @@
 
 (rf/defn show-logs-dialog
   {:events [:shake-event]}
-  [{:keys [db]}]
-  (when-not (:logging/dialog-shown? db)
```

Comment:
Hi @qfrank, Issue 1 and issue 4 are most probably closed due to removal of this check. For some reason :shake-event dispatched multiple times, and keeps opening the sheet.

![image](https://github.com/user-attachments/assets/335980ec-d0ef-4835-b9b4-e6ba5304023f)